custom/conf/app.ini.sample

```diff
@@ -579,8 +579,8 @@ DEFAULT_INTERVAL = 8h
 MIN_INTERVAL = 10m
 
 [api]
-; Enables /api/swagger, /api/v1/swagger etc. endpoints. True or false; default is true.
-ENABLE_SWAGGER_ENDPOINT = true
+; Enables Swagger. True or false; default is true.
+ENABLE_SWAGGER = true
```

Comment:
I am not against this change but it will need to be indicated as breaking. So we need to add the label kind/breaking on PR.